project_templates/technote_latex/testn-000/TESTN-000.tex

```diff
@@ -1,14 +1,12 @@
 \documentclass[DM,authoryear,toc]{lsstdoc}
 % lsstdoc documentation: https://lsst-texmf.lsst.io/lsstdoc.html
-
 \input{meta}
 
 % Package imports go here.
 
 % Local commands go here.
 
 % To add a short-form title:
-% \title[Short title]{Title}
```

Comment:
Shouldn't you be deleting the comment too if you're now dropping the example of a short-form title?

Reply:
ah yes done